pkg/schema/load/load_test.go

```diff
@@ -0,0 +1,96 @@
+package load
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/grafana/grafana"
+	"github.com/grafana/grafana/pkg/schema"
+	"github.com/stretchr/testify/require"
+)
+
+var p BaseLoadPaths = BaseLoadPaths{
+	BaseCueFS:       grafana.CoreSchema,
+	DistPluginCueFS: grafana.PluginSchema,
+}
+
+// Basic well-formedness tests on core scuemata.
+func TestScuemataBasics(t *testing.T) {
+	all := make(map[string]schema.VersionedCueSchema)
+
+	dash, err := BaseDashboardScuemata(p)
+	require.NoError(t, err, "error while loading base dashboard scuemata")
+	all["basedash"] = dash
+
+	ddash, err := DistDashboardScuemata(p)
+	require.NoError(t, err, "error while loading dist dashboard scuemata")
+	all["distdash"] = ddash
+
+	for set, sch := range all {
+		t.Run(set, func(t *testing.T) {
+			if sch == nil {
+				t.Error("scuemata linked to empty chain")
+			}
+
+			maj, min := sch.Version()
+			t.Run(fmt.Sprintf("%v.%v", maj, min), func(t *testing.T) {
+				cv := sch.CUE()
+				t.Run("Exists", func(t *testing.T) {
+					require.True(t, cv.Exists(), "cue value for schema does not exist")
+				})
+				t.Run("Validate", func(t *testing.T) {
+					require.NoError(t, cv.Validate(), "all schema should be valid with respect to basic CUE rules")
+				})
```

Comment:
Subtests (`t.Run`) should only be used for distinct test cases, not to group assertions.
```suggestion
				require.NoError(t, cv.Validate(), "all schema should be valid with respect to basic CUE rules")
```